lib/puppet/provider/cisco_vni/nxapi.rb

```diff
@@ -1,135 +0,0 @@
-# The NXAPI provider for cisco vni.
```

Comment:
@alok-aggarwal @dcheriancisco With these changes do we still need to keep `vni.rb` and related minitests under cisco-network-node-utils?

https://github.com/cisco/cisco-network-node-utils/blob/release_1.2.0/lib/cisco_node_utils/vni.rb